cmd/git/main.go

```diff
@@ -115,6 +116,10 @@ func Execute(ctx context.Context) error {
 	out, _ := git(ctx, "clone", "-h")
 	useNoTagsFlag = strings.Contains(out, "--no-tags")
 
+	// Check if Git CLI support --single-branch and therefore shallow clones using --depth
+	out, _ = git(ctx, "submodule", "-h")
```

Comment:
NIT: seems to make sense to group these help inspections in a separate function

Reply:
Yes, right, good idea. Can do that.